### ISSUE TITLE
`load` gives false error when document ends with `...`

### ISSUE THREAD

**Issue by adamvoss** [mulesoft-labs/yaml-ast-parser#20]
In the REPL:
```js
yaml = require('yaml-ast-parser')
result = yaml.load(`---
whatever: true
...`)
result.errors
```
outputs:
```js
[ YAMLException {
    name: 'YAMLException',
    reason: 'can not read an implicit mapping pair; a colon is missed',
    mark: 
     Mark {
       name: null,
       buffer: '---\nwhatever: true\n...\n\u0000',
       position: 19,
       line: 1,
       column: 0 },
    message: 'JS-YAML: can not read an implicit mapping pair; a colon is missed at line 2, column 1' } ]
```

However, the input document is valid and JS-YAML accepts it without error.

This issue becomes particularly pronounced when using `loadAll` to load multiple documents in a single stream, because this issue will occur many times.

**Comment by adamvoss** [comment]
I encounter this regularly with [YAML files that are processed to be webpages/documents using Jekyll](https://github.com/jekyll/jekyll/issues/6188).

**Comment by adamvoss** [comment]
Fixed by #24
